libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/LoginActivity.java

```diff
@@ -328,4 +383,17 @@ private void initAnalyticsManager() {
             analyticsManager.updateLoggingPrefs();
 	    }
     }
+
+    public class ChromeEventReceiver extends BroadcastReceiver {
+
+        @Override
+        public void onReceive(Context context, Intent intent) {
+            if (intent != null && intent.getAction() != null) {
+                final String action = intent.getAction();
+                if (ServerPickerActivity.CHANGE_SERVER_INTENT.equals(action)) {
```

Comment:
Switched to `Intent` mechanism for `Change Server` flow, since it's more robust and works in the `Chrome` case as well.